src/ai/backend/manager/config.py

```diff
@@ -80,7 +80,12 @@
                - window: "1m"  # average for duration
                # NOTE: To use "cuda.mem" criteria, user programs must use
                #       an incremental allocation strategy for CUDA memory.
-           - initial-grace_period: "30s"
+           + thresholds_check_condition:
+               - condition: "and" # logical operator
```

Comment:
Will there be more sub-items under `thresholds_check_condition`? If not, how about to set the condition under `thresholds-check-condition` instead of `thresholds_check_condition/condition`? Also, please use dash (`-`), not underscore (`_`) for consistency with other key names (such as `resource-thresholds` and `app-streaming-packet-timeout`.